core/config/mercury_config.go

```diff
@@ -16,8 +17,14 @@ type MercuryTLS interface {
 	CertFile() string
 }
 
+type MercuryTransmitter interface {
+	MaxTransmitQueueSize() uint32
```

Comment:
What about one of these formats, so the `Transmit` fields are grouped/sorted together?
```suggestion
	TransmitQueueMaxSize() uint32
```
```suggestion
	TransmitMaxQueueSize() uint32
```